definitions/Applications_CustomQuery.sqlx

```diff
@@ -67,7 +67,15 @@ SELECT
       FROM
         further_information_requests.created_at), EXTRACT(TIME
       FROM
-        started_at), MINUTE)) AS minutes_since_started_to_FIed
+        started_at), MINUTE)) AS minutes_since_started_to_FIed,
+  DATE_DIFF(further_information_requests.received_at, further_information_requests.requested_at,DAY) AS fi_days_waiting_to_receive,
+  DATE_DIFF(further_information_requests.reviewed_at, further_information_requests.received_at,DAY) AS fi_days_waiting_to_review,
+  DATE_DIFF(professional_standing.received_at, professional_standing.requested_at,DAY) AS ps_days_waiting_to_receive,
+  DATE_DIFF(professional_standing.reviewed_at, professional_standing.received_at,DAY) AS ps_days_waiting_to_review,
```

Comment:
With LoPS we have two levels of verification, first admins (and only if it's not approved by an admin) does it get reviewed by an assessor.

`verified_at` represents when the admin performs a verification, and `reviewed_at` represents when an admin does it.

I'd suggest something like this to handle both cases:

```suggestion
  DATE_DIFF(professional_standing.verified_at, professional_standing.received_at,DAY) AS ps_days_waiting_to_verify,
  DATE_DIFF(professional_standing.reviewed_at, professional_standing.verified_at,DAY) AS ps_days_waiting_to_review,
```